### PR TITLE
Adding references to the Extensions package to help promote it. (#4967)

### PR DESCRIPTION
**Pull request by surfnerd**
cherry-pick @mmattar's changes to mention extension docs.

### CODE REVIEW

File: com.unity.ml-agents.extensions/Documentation~/com.unity.ml-agents.extensions.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ directions (substituting `com.unity.ml-agents.extensions` for the package name).
 In Unity 2019.4 or later, open the Package Manager, hit the "+" button, and select "Add package from git URL".
 
 ![Package Manager git URL](https://github.com/Unity-Technologies/ml-agents/tree/release_13_docs/docs/images/unity_package_manager_git_url.png)
+
```
Comment (mmattar) [inline com.unity.ml-agents.extensions/Documentation~/com.unity.ml-agents.extensions.md:39]: In my PR, I also change L38... 🤷

Comment (surfnerd) [reply com.unity.ml-agents.extensions/Documentation~/com.unity.ml-agents.extensions.md:39]: fixed

File: com.unity.ml-agents/Documentation~/com.unity.ml-agents.md [hunk 2]
```diff
@@ -124,3 +129,4 @@ Please refer to "Information that is passively collected by Unity" in the
 [execution order of event functions]: https://docs.unity3d.com/Manual/ExecutionOrder.html
 [connect with us]: https://github.com/Unity-Technologies/ml-agents#community-and-feedback
 [ml-agents forum]: https://forum.unity.com/forums/ml-agents.453/
+[ML-Agents GitHub repo]: https://github.com/Unity-Technologies/ml-agents/blob/release_12_docs/com.unity.ml-agents.extensions
```
Comment (mmattar) [inline com.unity.ml-agents/Documentation~/com.unity.ml-agents.md:132]: Should this be Release 13?

Comment (chriselion) [reply com.unity.ml-agents/Documentation~/com.unity.ml-agents.md:132]: @surfnerd Any idea why the pre-commit script missed this?

Comment (chriselion) [reply com.unity.ml-agents/Documentation~/com.unity.ml-agents.md:132]: Never mind, this was branched off the release branch before that was merged.

Comment (chriselion) [inline com.unity.ml-agents/Documentation~/com.unity.ml-agents.md:132]:
```suggestion
[ML-Agents GitHub repo]: https://github.com/Unity-Technologies/ml-agents/blob/release_13_docs/com.unity.ml-agents.extensions
```

**Review by mmattar — APPROVED** [review]

**Pull request merged**